File: evancolwell.rs
fn main(){
    let mut x = 1;
    while x < 100 {
        if x % 3 == 0 && x % 5 != 0{
            println!("Fizz");
        } else if x % 3 != 0 && x % 5 == 0 {
            println!("Buzz");
        } else if x % 3 == 0 && x % 5 == 0 {
            println!("FizzBuzz");
        } else {
            println!("{}", x);
        }
        x += 1;
    }
}

File: evancolwell.py


def main():
    for x in range(1, 101):
        if (x%3 == 0 and x%5 != 0):
            print("Fizz\n")
        elif(x%3 != 0 and x%5 == 0):
            print("Buzz\n")
        elif(x%3 == 0 and x%5 == 0):
            print("FizzBuzz\n")
        else:
            print(str(x)+"\n")

main()
